src/utils/constants/formSteps.ts

```diff
@@ -22,18 +22,18 @@ export const formSteps = [
     id: 'Step 3',
     name: 'Opportunities',
     fields: [
-      'volunteers',
-      'vol_reqs',
-      'vol_coor',
-      'vol_coor_tel',
-      'donation',
-      'don_ex',
-      'pickup',
-      'pickup_loc',
-      'don_coor',
-      'don_coor_tel',
-      'recommendation',
-      'recommendations_contact',
+      'volunteerFields.volunteers',
+      'volunteerFields.vol_reqs',
+      'volunteerFields.vol_coor',
+      'volunteerFields.vol_coor_tel',
+      'donationFields.donation',
+      'donationFields.don_ex',
+      'donationFields.pickup',
+      'donationFields.pickup_loc',
+      'donationFields.don_coor',
+      'donationFields.don_coor_tel',
+      'recommendationFields.recommendation',
+      'recommendationFields.recommendations_contact',
```

Comment:
same with all of these subfields. should be able to be consolidated into "volunteerFields", etc.